updatecli/updatecli.d/klipper-lb.yaml

```diff
@@ -12,17 +12,7 @@ scms:
       repository: "{{ .k3s.repo }}"
       branch: "{{ .k3s.branch }}"
       commitmessage:
-        title: "Bump Klipper version"
-  klipper-helm:
-    kind: "github"
-    spec:
-      user: "{{ .github.user }}"
-      email: "{{ .github.email }}"
-      username: "{{ .github.username }}"
-      token: "{{ requiredEnv .github.token }}"
-      owner: "{{ .k3s.org }}"
-      repository: "{{ .klipper_helm.repo }}"
-      branch: "{{ .klipper_helm.branch }}"
+        title: "Bump Klipper LB version"
```

Comment:
Out of curiosity, is the generated commit message too verbose (should we always set a commit title?)

Reply:
The default commit message will look like this - https://github.com/macedogm/k3s/pull/9/commits/20ab3e064731db3d1b49f2dcb7715f20bd71680b, in case we don't add a new one.

I guess this is an improvement for UpdateCLI, because they could drop the entire `<temp-folder>/<repo_name>` from the commit message - `"/var/folders/gs/8nqtp16979lbq32ycbcgnpcm0000gp/T/updatecli/github/macedogm/k3s/`.